src/core/database/wallet_memory.zig

```diff
@@ -34,34 +36,34 @@ pub const WalletMemoryDatabase = struct {
         keyset_counter: std.AutoHashMap(nuts.Id, u32),
         nostr_last_checked: std.AutoHashMap(secp256k1.PublicKey, u32),
     ) !WalletMemoryDatabase {
-        var _mint_quotes = std.AutoHashMap([16]u8, MintQuote).init(allocator);
+        var _mint_quotes = std.AutoHashMap(zul.UUID, MintQuote).init(allocator);
         errdefer _mint_quotes.deinit();
 
         for (mint_quotes) |q| {
             try _mint_quotes.put(q.id, q);
         }
 
-        var _melt_quotes = std.AutoHashMap([16]u8, MeltQuote).init(allocator);
+        var _melt_quotes = std.AutoHashMap(zul.UUID, MeltQuote).init(allocator);
         errdefer _melt_quotes.deinit();
 
         for (melt_quotes) |q| {
             try _melt_quotes.put(q.id, q);
         }
 
-        var _mint_keys = std.AutoHashMap(nuts.Id, nuts.nut01.Keys);
+        var _mint_keys = std.AutoHashMap(nuts.Id, nuts.nut01.Keys).init(allocator);
         errdefer _mint_keys.deinit();
```

Comment:
```suggestion
        errdefer _mint_keys.deinit();
        errdefer {
            var it = _mint_keys.valueIterator();
            while (it.next()) |k| {
                k.deinit();
            }
        }
```
because if after throws error, we have data inside keys, that was allocated, but not deallocated